tutorials/29_Serializing_Pipelines.ipynb

```diff
@@ -0,0 +1,413 @@
+{
```

Comment:
<p>It would be great to add one sentence explaining why serialization is useful since IMO it's non obvious. It's pretty easy to pass around chunks of Python code programmatically? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/272/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>